DQM/EcalMonitorDbModule/plugins/EcalDQMStatusWriter.cc

```diff
@@ -30,25 +29,27 @@ void EcalDQMStatusWriter::beginRun(edm::Run const &_run, edm::EventSetup const &
   statusManager.writeToObj(channelStatus_, towerStatus_);
 }
 
+void EcalDQMStatusWriter::endRun(edm::Run const &_run, edm::EventSetup const &_es) {}
+
 void EcalDQMStatusWriter::analyze(edm::Event const &, edm::EventSetup const &_es) {
   cond::service::PoolDBOutputService &dbOutput(*edm::Service<cond::service::PoolDBOutputService>());
   if (firstRun_ == dbOutput.endOfTime())
     return;
 
-  dbOutput.writeOne(&channelStatus_, firstRun_, "EcalDQMChannelStatusRcd");
-  dbOutput.writeOne(&towerStatus_, firstRun_, "EcalDQMTowerStatusRcd");
+  dbOutput.writeOne(channelStatus_, firstRun_, "EcalDQMChannelStatusRcd");
```

Comment:
Hi @ggovi 
The changes that I needed to make in order to get `conddb dump` to work are these two lines.

Reply:
@thomreis Can you please dump the error that you get with conddb dump?

Reply:
Sure. I have also sent it in an email to you last week already.
```
$ conddb --db mask-ECAL.db dump a68ed854e09eb04addd68ec350b820a4e171ca66
[2021-09-29 16:03:38,098] INFO: Connecting to mask-ECAL.db [sqlite:///mask-ECAL.db]
[2021-09-29 16:03:38,150] INFO: Found payload of type EcalCondTowerObjectContainer<EcalDQMStatusCode>*
[2021-09-29 16:03:39,303] WARNING: No XML converter for payload class EcalCondTowerObjectContainer<EcalDQMStatusCode>* found in the built-in library.
```

Reply:
@thomreis thanks, i'll investigate.